public/stylesheets/account.css

```diff
@@ -105,3 +117,67 @@ main {
   margin-top: -3px;
   font-size: 9px;
 }
+
+@media (max-width: 1024px) {
+  body {
+    font-size: 10px;
+  }
+  .profile-img {
+    width: 95px;
+    height: 95px;
+  }
+  .sidebar-tab {
+    font-size: 12px;
+    padding: 10px 10px;
+  }
+  .account-first-row {
+    min-height: 131px;
+  }
+  main {
+    padding: 70px 8% 70px 5%;
+  }
+}
+
+@media (max-width: 576px) {
+  .profile-img {
+    width: 130px;
+    height: 130px;
+  }
+  .main-box {
+    display: block;
+  }
+  .sidebar,
+  .info-box {
+    width: 100%;
+  }
+  .sidebar {
+    border-right: none;
+    margin-bottom: 50px;
+  }
+  .sidebar-tab {
+    text-align: center;
+    display: block;
+  }
+  main {
+    padding: 70px 15px;
+  }
+  .main-box {
+    width: 90%;
+  }
+  .form-section-label {
+    font-size: 15px;
+  }
+  .profile-form-input,
+  #bio {
+    font-size: 10px;
+  }
+  .form-input-label {
+    font-size: 11px;
+  }
+  .img-description {
+    font-size: 8px;
```

Comment:
I would recommend to not go below `12px` when it comes to font size. You wouldn't want the user to squint their eyes when using your app.